server/db/mysql/schema.sql

```diff
@@ -1,3 +1,13 @@
+# THIS SCHEMA FILE IS FOR REFERENCE/DOCUMENTATION ONLY!
+# DO NOT USE IT TO INITIALIZE THE DATABASE.
+# Read installation instructions first.
+
+# The following line will produce an intentional error.
+
+'READ INSTALLATION INSTRUCTIONS!';
```

Comment:
Perhaps, add a link here?

Reply:
I thought about it and decided against it: we will forget about it and it will get stale. It's just a forcing function to make people search for answers.